person/models.py

```diff
@@ -157,15 +157,17 @@ class GodFather(Person):
                                  verbose_name='Cantidad', validators=[MinValueValidator(1)])
     frequency = models.CharField(
         max_length=20, choices=FREQUENCY, verbose_name='Frecuencia de pago')
-    seniority = models.DateField(verbose_name='Antigüedad')
+    seniority = models.DateTimeField(verbose_name='Antigüedad')
     notes = models.TextField(blank=True, verbose_name='Observaciones')
     status = models.CharField(
         max_length=20, choices=STATUS, verbose_name='Estado')
     slug = models.SlugField(max_length=200, unique=True, editable=False)
 
     def save(self, *args, **kwargs):
-        self.slug = slugify(self.name + ' ' + self.surname)
-
+        self.slug = slugify(str(self.postal_code) + ' '+self.name + ' ' + self.surname)
+        if self.seniority < self.birth_date:
+            raise ValidationErr(
+                "The seniority date can't be before the birth date")
```

Comment:
I think It's better to show message error in spanish. Our client may understand only spanish and not know english.

Reply:
Ok. It has neen changed :)